app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -7,4 +7,26 @@ def retire_now(requester = nil)
     update_attributes(:retirement_requester => requester)
     finish_retirement
   end
+
+  # Intend to be called by UI to display stdout. Therefore the error message directly returned
+  # instead of raising an exception.
+  def raw_stdout_via_worker(userid = User.current_user, format = 'txt')
+    unless MiqRegion.my_region.role_active?("embedded_ansible")
+      return "Cannot get standard output of this playbook because the embedded Ansible role is not enabled"
+    end
+
+    options = {:userid => userid, :action => 'ansible_stdout'}
```

Comment:
Could be something wrong here.. the resulting task (accessed via the API) gives me...

`userid: "#<User:0x00007fdd65fc6c38>"`